api-admin/src/main/java/bio/terra/pearl/api/admin/controller/GlobalExceptionHandler.java

```diff
@@ -72,6 +77,12 @@ public ResponseEntity<ErrorReport> internalErrorExceptionHandler(Exception ex) {
     return buildErrorReport(ex, HttpStatus.INTERNAL_SERVER_ERROR, request);
   }
 
+  @ExceptionHandler({AddressValidationException.class})
+  public ResponseEntity<ErrorReport> addressValidationExceptionHandler(
+      AddressValidationException ex) {
+    return buildErrorReport(ex, ex.getHttpStatusCode(), request);
+  }
```

Comment:
Is this the right way to add error handling for this exception? It primarily throws 502s, which I think but am not sure if it is the correct code to return.